recipes/knot/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: knot
+  version: "1.0.0"
+
+source:
+  fn: KNOT-1.0.0.tgz
+  url: http://mathbio.nimr.mrc.ac.uk/download/KNOT/KNOT-1.0.0.tgz
+  md5: 2556876109bae506fe171a45902db0b6
+
+build:
+  number: 1
+
+test:
+  commands:
+    - which knot
```

Comment:
Sorry, I didn't see your comment. What about running it without args and use grep to check for an expected error message.